deployment/flexible_api_server.py

```
@@ -0,0 +1,564 @@
+        logger.info(f"🔄 Initializing {self.deployment_type.value} deployment...")
+
+        if self.deployment_type == DeploymentType.SERVERLESS:
+            self._initialize_serverless()
+        elif self.deployment_type == DeploymentType.ENDPOINT:
+            self._initialize_endpoint()  
+        elif self.deployment_type == DeploymentType.LOCAL:
+            self._initialize_local()
+
+        logger.info("✅ Initialization complete!")
+
+    def _initialize_serverless(self):
```

Ruff (0.12.2)

Missing return type annotation for private function `_initialize_serverless`

Add return type annotation: `None`

(ANN202)